package/gluon-mesh-batman-adv/luasrc/lib/gluon/upgrade/320-gluon-mesh-batman-adv-client-bridge

```diff
@@ -34,12 +34,13 @@ uci:section('network', 'route6', 'local_node_route6', {
 
 uci:save('network')
 
+local networks = uci:get_list('firewall', 'mesh', 'network')
+util.add_to_set(networks, 'client')
+uci:set_list('firewall', 'mesh', 'network', networks)
 
-uci:section('firewall', 'zone', 'client', {
-	input = 'ACCEPT',
-	output = 'ACCEPT',
-	forward = 'REJECT',
-})
+local networks = uci:get_list('firewall', 'drop', 'network')
+util.remove_from_set(networks, 'client')
+uci:set_list('firewall', 'drop', 'network', networks)
```

Comment:
I don't think using the add_to_set/remove_from_set helpers is necessary here, as we know the exact list of interfaces for each zone.

Reply:
I disagree. We only know the interfaces for each zone if the user didn't fiddle with it. I feel that using the helpers makes the code more stable, therefore we should keep it.

Reply:
The user can't fiddle with this, the zones will be overwritten completely on each upgrade. The only reason to have the helpers would be to make it more flexible for the addition of other packages affecting the firewall configuration (which would be easier to implement by running another script afterwards). But I don't care enough for either solution, we can keep it as is.